services/app-web/src/pages/CMSDashboard/CMSDashboard.tsx

```diff
@@ -103,11 +103,12 @@ export const CMSDashboard = (): React.ReactElement => {
             }
 
             // Calculate proper submission data to display in each row
-            // if UNLOCKED package, form data could be in progress of being edited, show the static form data from the last submitted revision
-            // also, use lastUpdated from unlockData (if unlocked) or last submit (if initial submission)
-            // otherwise, use data from current revision for Submitted or Resubmitted packages
+            // if Unlocked package, form data could be in progress of being edited, show the static form data from the last submitted revision
+            // otherwise, use current data for Submitted or Resubmitted packages
             let packageDataToDisplay = currentFormData
-            let lastUpdated = currentFormData.updatedAt
+            let lastUpdated =
```

Comment:
Consider making a utility function to compare a set of dates and pull most recent as last updated